listenbrainz_spark/stats/tests/test_init.py

```diff
@@ -0,0 +1,37 @@
+	#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Dec 15 20:12:56 2019
+
+@author: sarthak
+"""
+
+import datetime
+from listenbrainz_spark import utils
+from pyspark.sql import Row
+from listenbrainz_spark import stats
+from listenbrainz_spark.tests import SparkTestCase
+
+
+class InitTestCase(SparkTestCase):
+
+    def test_replace_days(self):
+        self.assertEqual(stats.replace_days(datetime.datetime(2019,5,12),13),datetime.datetime(2019,5,13,0,0))
+        
+    def test_adjust_months(self):
+        self.assertEqual(stats.adjust_months(datetime.datetime(2019,5,12),3,True),datetime.datetime(2019,2,12,0,0))
+        
+    def test_ajust_days(self):
+        self.assertEqual(stats.adjust_days(datetime.datetime(2019,5,12),3,True),datetime.datetime(2019,5,9,0,0))
+        
+    def test_run_query(self):
+        df = utils.create_dataframe(Row(column1=1, column2=2), schema=None)
+        df.createTempView("table")
+        new_df=stats.run_query("""
+      SELECT *
+        FROM table
+     """)
+        self.assertEqual(new_df.count(),df.count())
+        
+        
+        
```

Comment:
There are too many new lines at the end of your file. You can change settings of your editor to ensure one trailing newline at the end of the file. 
[Why newlines are required](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)